src/use-image.hook.js

```diff
@@ -29,6 +29,7 @@ export function useImage({processImgElement = noop, fileBlobToUrl = defaultFileB
 
   function handleImageElement(imgElement) {
     imgElement.style.cursor = 'pointer'
+    imgElement.style.maxWidth = '100%'
```

Comment:
I can't remember, if the `img` hits this max will it auto-proportion the height, squish the image horizontally, or clip the image?  I'm guessing squish, but could be wrong.

Reply:
When you set the width but not the height, it makes the height be the correct height to maintain the aspect ratio